tutor/commands/upgrade/k8s.py

```diff
@@ -165,6 +165,26 @@ def upgrade_from_olive(context: Context, config: Config) -> None:
     upgrade_mongodb(config, "4.2.17", "4.2")
     upgrade_mongodb(config, "4.4.22", "4.4")
 
+    upgrade_to_redwood_onwards, _ = (
+        common_upgrade.verify_tutor_version_for_mysql_upgrade(config)
+    )
+
+    if not upgrade_to_redwood_onwards:
+        return
+
+    # We need to first revert MySQL back to v8.1 to build the data dictionary on it
+    # And also to update the authentication plugin as it is disabled on v8.4
```

Comment:
nit: we can remove this comment as the message on the next lines conveys the same thing.

Reply:
Removed the comments.